### DISCUSSION TITLE
Welcome to lmt_toolkit_analysis Discussions!

### DISCUSSION THREAD

**Discussion by ntorquet** [ntorquet/lmt_toolkit_analysis#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LWitter** [comment]
How do I use this package? I have several LMT experiments, but can't figure out how to use your code to do anything...

**Reply by ntorquet** [reply]
Hi Laurens,

LMT-toolkit is not a Python package, it is a complete application coded in both Python and Vuejs (javascript).
To use it, you need to install it as described in the step-by-step guide in the readme [here](https://github.com/ntorquet/lmt_toolkit_analysis#installation)

If you have any problems with a step, please contact me and I will help you.

If you just want to run a Python code (which means that you have Python coding skills), you can directly use the lmt-analysis code available [here](https://github.com/fdechaumont/lmt-analysis)
This is the same code used by LMT-toolkit for analysis.

LMT-toolkit allows you to perform quality control and analysis without writing a single line of code. Installing the application is not trivial, however, for which I apologize.

Best,

Nicolas
